recipe/meta.yaml

```diff
@@ -49,7 +60,7 @@ outputs:
         - export CUDA_PATH=$PREFIX/targets/sbsa-linux/           # [cuda_compiler_version == "None" and aarch64]
         - set CUDA_PATH=%PREFIX%\Library                         # [cuda_compiler_version == "None" and win64]
         - >-
-          {{ PYTHON }} -m pip install --no-deps --no-build-isolation -v .
+          {{ PYTHON }} -m pip install --no-deps --no-build-isolation -v .{{ extras }}
```

Comment:
No. We have provided `--no-deps`, so no extras are installed. Only the requirement of the extras is added to the wheel metadata... I actually didn't check if adding an extra modifies the metadata, but it should(?) otherwise `pip check` would be useless for packages installed with extras.

Reply:
This is a cool idea of combining extras with --no-deps.